src/VolleyManagement.WebClient/spa/app/Components/Schedule/schedule.component.html

```diff
@@ -8,95 +8,81 @@
+                  <div *ngIf="isVideoLinkVisible(gameResult)" class="videoIconCell">
                     <a href="{{gameResult.UrlToGameVideo}}" title="Перейти к видео" class="videoLinkIcon">
                       📹
                     </a>
-                  </td>
-
-                  <td class="teamname" colspan="5">
-                    {{gameResult.HomeTeamName}}
-                  </td>
-
-                  <td class="teamscore">
-                    {{gameResult.Result.TotalScore.Home}}
-                  </td>
-
-                </tr>
-
-                <tr class="team-sch" [class.win]="gameResult.Result.TotalScore.Home < gameResult.Result.TotalScore.Away">
-
-                  <td class="teamname" colspan="5">
-                    {{ gameResult.AwayTeamName }}
-                  </td>
-
-                  <td class="teamscore">
-                    {{ gameResult.Result.TotalScore.Away }}
-                  </td>
+                  </div>
+                </td>
+
+                <!--Main content: Team names, results-->
+
+                <td class="teamname" colspan="5" [class.win]="isHomeWinner(gameResult)">
+                  {{gameResult.HomeTeamName}}
+                </td>
+                <td class="teamscore" *ngIf="isGameResultVisible(gameResult)" [class.win]="isHomeWinner(gameResult)">
+                  {{gameResult.Result.TotalScore.Home}}
+                </td>
+
+                <!--Right column: score, date, free day-->
+                <td *ngIf="isGameDateVisible(gameResult)" rowspan="2">
+                  {{gameResult.GameDate| localDate | date: 'HH:mm'}}
+                </td>
+                <td *ngIf="isFreeDayVisible(gameResult)">
+                  вых.
+                </td>
+              </tr>
+
+              <tr>
+                <td class="teamname" colspan="5" [class.win]="isAwayWinner(gameResult)">
+                  {{gameResult.AwayTeamName}}
+                </td>
+                <td class="teamscore" *ngIf="isGameResultVisible(gameResult)" [class.win]="isAwayWinner(gameResult)">
+                  {{gameResult.Result.TotalScore.Away}}
+                </td>
+
+                <tr class="sets">
+                  <ng-template [ngIf]="isGameResultVisible(gameResult)">
+                    <td class="set" *ngFor="let setScore of gameResult.Result.SetScores">
+
+                      <ng-template [ngIf]="!setScore.IsEmpty && !gameResult.Result.IsTechnicalDefeat">
+                        <div>
+                          {{setScore.Home}} : {{setScore.Away}}
+                        </div>
+                      </ng-template>
+
+                      <ng-template [ngIf]="!setScore.IsEmpty && gameResult.Result.IsTechnicalDefeat">
+                        <div>
+                          <span>техн.</span>
+                        </div>
+                      </ng-template>
+
+                      <ng-template [ngIf]="setScore.IsEmpty">
```

Comment:
What is the point of this block? If I understand css class clear it is hidden always.

Reply:
hm... yes. I haven't looked at it - I copied it from old code. I'll try to remove it